evol/population.py

```diff
@@ -279,7 +279,7 @@ def callback(self, callback_function: Callable[['BasePopulation'], None],
         :return:
         """
         self.evaluate(lazy=True)
```

Comment:
but we're still evaluating the entire population here right? or is the copying the expensive part?

Reply:
Those evaluations may already be present (hence the `lazy`), and can be used in the future.